app/release-notes/use-release-notes.ts

```diff
@@ -1,29 +1,46 @@
-import {getRepo} from "app/core/utils/get-repo"
-import {useEffect, useState} from "react"
+import {metaClient} from "app/ipc/meta"
+import {useEffect, useState, version} from "react"
 
-export function useReleaseNotes(version) {
+async function fetchNotes() {
```

Comment:
Shouldn't you still be passing version in here? Or is version imported from react the set Version?

Reply:
Nope, you're correct. That was a mistake.

Reply:
I updated the test to catch this and fixed it.